src/rules/function-max-empty-lines/__test__/index.js

```diff
@@ -0,0 +1,209 @@
+    message: messages.rejected,
+    line: 3,
+    column: 2,
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n\r\n\r\n1\r\n); }",
+    message: messages.rejected,
+    line: 3,
+    column: 3,
+  }, {
+    code: "a { transform: translate(\n1\n,\n1\n\n\n); }",
+    message: messages.rejected,
+    line: 4,
+    column: 2,
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n1\r\n\r\n\r\n); }",
+    message: messages.rejected,
+    line: 4,
+    column: 3,
+  } ],
+})
+
+testRule(rule, {
+  ruleName,
+  config: [2],
+
+  accept: [ {
+    code: "a { transform: translate(\n1\n,\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n\n1\n,\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n\r\n1\r\n,\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n\n,\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n\r\n,\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n,\n\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n,\n1\n\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n1\r\n\r\n); }",
+  }, {
+    code: "a { transform: translate(\n\n\n1\n,\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n\r\n\r\n1\r\n,\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n\n\n,\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n\r\n\r\n,\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n,\n\n\n1\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n\r\n\r\n1\r\n); }",
+  }, {
+    code: "a { transform: translate(\n1\n,\n1\n\n\n); }",
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n1\r\n\r\n\r\n); }",
+  } ],
+
+  reject: [ {
+    code: "a { transform: translate(\n\n\n\n1\n,\n1\n); }",
+    message: messages.rejected,
+    line: 1,
+    column: 26,
+  }, {
+    code: "a { transform: translate(\r\n\r\n\r\n\r\n1\r\n,\r\n1\r\n); }",
+    message: messages.rejected,
+    line: 1,
+    column: 27,
+  }, {
+    code: "a { transform: translate(\n1\n\n\n\n,\n1\n); }",
+    message: messages.rejected,
+    line: 2,
+    column: 2,
+  }, {
+    code: "a { transform: translate(\r\n1\r\n\r\n\r\n\r\n,\r\n1\r\n); }",
+    message: messages.rejected,
+    line: 2,
+    column: 3,
+  }, {
+    code: "a { transform: translate(\n1\n,\n\n\n\n1\n); }",
+    message: messages.rejected,
+    line: 3,
+    column: 2,
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n\r\n\r\n\r\n1\r\n); }",
+    message: messages.rejected,
+    line: 3,
+    column: 3,
+  }, {
+    code: "a { transform: translate(\n1\n,\n1\n\n\n\n); }",
+    message: messages.rejected,
+    line: 4,
+    column: 2,
+  }, {
+    code: "a { transform: translate(\r\n1\r\n,\r\n1\r\n\r\n\r\n\r\n); }",
+    message: messages.rejected,
+    line: 4,
+    column: 3,
+  } ],
+})
```

Comment:
All the tests are testing functions. I would like to see at least a couple of tests testing against false positives outside of functions. Like the example I gave. Just to ensure that the rule is accurate targeting *only* the newlines it's supposed to target, not other newlines in declarations or values.